config/database.yml

```diff
@@ -23,3 +23,7 @@ test:
 production:
   <<: *default
   database: db/production.sqlite3
+
+staging:
+  <<: *default
+  database: db/staging.sqlite3
```

Comment:
https://github.com/andela-iamadi/zhishi-backend/blob/master/config/database.yml#L25 seems to be pointing to sqlite3 or is there something I am missing?